tests/test_ixmp4.py

```diff
@@ -20,16 +23,38 @@ def test_to_ixmp4_missing_unit_raises(test_df_year):
         test_df_year.to_ixmp4(platform=platform)
 
 
-def test_ixmp4_integration(test_df):
-    """Write an IamDataFrame to the platform"""
+def test_ixmp4_time_not_implemented(test_df):
+    """Writing an IamDataFrame with datetime-data is not implemented"""
     platform = Platform(_backend=SqliteTestBackend())
     platform.regions.create(name="World", hierarchy="common")
     platform.units.create(name="EJ/yr")
 
-    if test_df.time_domain == "year":
-        test_df.to_ixmp4(platform=platform)
-    else:
+    if test_df.time_domain != "year":
         with pytest.raises(NotImplementedError):
             test_df.to_ixmp4(platform=platform)
+        # pass
+
+
+def test_ixmp4_integration(test_df_year):
```

Comment:
It might make sense to split this test into two parts:
1. Reading only the default
2. Reading all scenarios

Reply:
I'm not sure yet how I want to structure these tests when adding different filters, so prefer to leave as is for now.

Reply:
Ok, fine to leave then.